### DISCUSSION TITLE
How/When to log server readiness?

### DISCUSSION THREAD

**Discussion by Zapeth** [actix/actix-web#2204] [Q&A]
I'd like to log/print a message when the server is ready to receive connections, kind of like the `listening` event in nodejs:
```javascript
server.listen(() => { console.log('listening on', server.address()); });
```

How would an equivalent look/work with actix-web?

**Comment by fakeshadow** [comment]
https://github.com/actix/examples/blob/bb4f97ba4186f6931fcca170339c59e2f2be79a6/basics/basics/src/main.rs#L67

**Reply by Zapeth** [reply]
Thanks, that works.

Is it also possible to do the same thing on the user side (meaning the user of the crate), without having to rely on internal log messages?
